lib/cdo/data/logging/rake_task_event_logger.rb

```diff
@@ -0,0 +1,76 @@
+require 'action_view'
+require lib_dir 'cdo/data/logging/timed_task_with_logging'
+class RakeTaskEventLogger
+  STUDY_TABLE = 'rake_performance'.freeze
+  CURRENT_LOGGING_VERSION = 'v0'.freeze
+
+  def initialize(rake_task)
+    @start_time = 0
+    @end_time = 0
+    @rake_task = rake_task
+    @enabled = true
+    #@enabled = !([:development, :test].include?(rack_env))
+  end
+
+  def start_task_logging
+    puts "hello"
```

Comment:
is a `puts` necessary here? I'd suggest deleting or editing to say something more useful like "starting rake task logging"